cmd/siad/main.go

```diff
@@ -194,6 +195,7 @@ func main() {
 	root.Flags().StringVarP(&globalConfig.Siad.APIaddr, "api-addr", "", "localhost:9980", "which host:port the API server listens on")
 	root.Flags().StringVarP(&globalConfig.Siad.SiaDir, "sia-directory", "d", "", "location of the sia directory")
 	root.Flags().BoolVarP(&globalConfig.Siad.NoBootstrap, "no-bootstrap", "", false, "disable bootstrapping on this run")
+	root.Flags().BoolVarP(&globalConfig.Siad.UseUPNP, "upnp", "", true, "use UPnP for port forwarding and external IP discovery")
```

Comment:
I also prefer it disabled by default, but should this default to true? There could be users relying on the existing functionality and unable to change it when using Sia-UI.

Reply:
the default is `true`, so UPnP will remain enabled by default

Reply:
Oh you’re right, bad comprehension on my part 🤦‍♂️